website/content/middleware/timeout.md

```diff
@@ -58,6 +72,64 @@ TimeoutErrorHandlerWithContext func(error, echo.Context) error
 DefaultTimeoutConfig = TimeoutConfig{
     Skipper:      DefaultSkipper,
     Timeout:      0,
-    ErrorHandler: nil,
+    ErrorMessage: "",
+}
+```
+
+## Alternatively handle timeouts in handlers
```

Comment:
I'm sure I'm missing something, because for me this is what the Timeout middleware is actually doing. Could you please help me to understand why this is better vs using the Timeout middleware?

Reply:
In my opinion handling work in background is not webserver responsibility and should be implemented explicitly in business domain (i.e. by developer). This ensures that people will not set `e.Use(middleware.Timeout())` mindlessly without thinking what and why they actually do it.